### PR TITLE
Z-Wave: Update screenshots

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Z-Wave: Update screenshots
- to reflect current UI

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/images/integrations/z-wave/z-wave-device-info.png
Comment (AlCalzone) [inline source/images/integrations/z-wave/z-wave-device-info.png:1]: The "menu" tooltip under the cursor is a bit unfortunate, other than that it's fine.

**Review by AlCalzone — APPROVED** [review]

File: source/images/integrations/z-wave/z-wave-device-info.png
Comment (c0ffeeca7) [reply source/images/integrations/z-wave/z-wave-device-info.png:1]: Well spotted. Thank you!

**Comment by MartinHjelmare** [comment]
What's different in the first image changed? I can't find the five things that are different. 😄

**Comment by c0ffeeca7** [comment]
> What's different in the first image changed? I can't find the five things that are different. smile

Not sure I can get 5, but let me try :laughing: 

1. Under **Device info**, there is now an icon. 
1. Under **Sensors**, the states are no longer **Unknown**
1. Under **Sensors**, there is an additional state
1. Under **Sensors**, the more of the strings are readable

Agreed, it is not a critical change. But I came across this by chance and thought it is not nice to show a screenshot with states **Unknown** and unreadable strings.

A clerical change, but I believe it improves quality.

**Review by MartinHjelmare — APPROVED** [review]
Good! Thanks!

**Pull request merged**